pkg/controller/utils/clientwrapper.go

```diff
@@ -29,15 +29,17 @@ func init() {
 }
 
 // NewClientWithMetricsOrDie creates a new controller-runtime client with a wrapper which increments
-// metrics for requests by controller name, HTTP method, and URL path. The client will re-use the
-// managers cache. This should be used in all Hive controllers.
+// metrics for requests by controller name, HTTP method, URL path, and whether or not the request was
+// to a remote cluster.. The client will re-use the managers cache. This should be used in
+// all Hive controllers.
```

Comment:
Moving to `pkg/controller/utils` is fine as `The client method should be used in all Hive controllers.`